MATLAB/utl/utl_orbrate.m

```diff
@@ -0,0 +1,8 @@
+function n = utl_orbrate(a)
+% Calculates the orbital rate of the sattelite assuming a circular orbit
```

Comment:
Add what units a and n are to the description.

Reply:
Yeah good point - MKS for everything.